model/image.go

```diff
@@ -31,8 +32,19 @@ import (
 
 const (
 	ArtifactFileSuffix = ".mender"
+	runeDollar         = '\uFF04'
+	runeDot            = '\uFF0E'
 )
 
+var (
+	StorageKeyImageProvidesRootFSChecksum = "meta_artifact.provides." +
+		GetProvidesKeyReplacer().Replace("rootfs-image.checksum")
+	StorageKeyImageProvidesRootFSVersion = "meta_artifact.provides." +
+		GetProvidesKeyReplacer().Replace("rootfs-image.version")
+)
```

Comment:
Not tested, but you get the idea... why initializing the replacer each time you use it?

```suggestion
var (
        providesKeyReplacer = GetProvidesKeyReplacer()
	StorageKeyImageProvidesRootFSChecksum = "meta_artifact.provides." +
		providesKeyReplacer.Replace("rootfs-image.checksum")
	StorageKeyImageProvidesRootFSVersion = "meta_artifact.provides." +
		providesKeyReplacer.Replace("rootfs-image.version")
)
```

Reply:
done